.env.example

```
@@ -1,11 +1,4 @@
-DODO_API_KEY_LIVE=your_live_api_key_here # obtained from Dodo Payments Dashboard in live mode
-DODO_PAYMENTS_WEBHOOK_KEY=your_webhook_key_here # obtained from Dodo Payments Dashboard
-
-
-# Environment
-NODE_ENV=development
-
-# Base URL for API requests
-NEXT_PUBLIC_BASE_URL=http://localhost:3000
+DODO_PAYMENTS_API_KEY=your_api_key_here
+DODO_PAYMENTS_WEBHOOK_KEY=your_webhook_key_here
+DODO_PAYMENTS_RETURN_URL=http://localhost:3000
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DODO_PAYMENTS_RETURN_URL key should go before the DODO_PAYMENTS_WEBHOOK_KEY key

(UnorderedKey)